cmd/synthetic-monitoring-agent/main.go

```diff
@@ -209,6 +212,15 @@ func run(args []string, stdout io.Writer) error {
 		zl.Info().Msg("disabling k6 checks")
 	}
 
+	var usageReporter usage.Reporter
+	switch {
+	case config.DisableUsageReports:
+		usageReporter = usage.NewNoOPReporter()
+	default:
+		usageReporter = usage.NewHTTPReporter(usage.ProdStatsEndpoint)
```

Comment:
ah, interesting...

Does this mean that the agents in dev clusters will be reporting to the prod instance of this?